config/routes.rb

```diff
@@ -16,6 +16,13 @@
     end
     resources :products, only: [:show]
     resources :sessions, only: [:new, :create, :destroy]
-    resources :orders, only: [:new, :create]
+    resources :orders, only: [:new, :create, :index] do
+      collection do
+        put "update_status/:id", to: "orders#update_status", as: "update_status"
+      end
```

Comment:
ở đâu có param id của orders phải ko em? nếu là id của order thì nên dùng member nha, với lại nó đang thuộc vể orders rồi nên ko cần phải chỉ rõ controller#action nếu giống nhau giữa end_point url và action ha
```suggestion
      member do
        put :update_status
      end
```

Reply:
dạ vâng chị